Gemfile.lock

```diff
@@ -258,7 +258,7 @@ GEM
       eventmachine (>= 0.12.9)
       http_parser.rb (~> 0.5.3)
     erubis (2.7.0)
-    eventmachine (1.0.8)
+    eventmachine (1.2.3)
```

Comment:
I would simply add that the dependents of this gem do not require that version so the reader will understand why we did this